src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -224,50 +257,57 @@ const DataSubview: FunctionComponent<Props> = ({
           />
           <StyledDiv>Selected </StyledDiv>
           <Divider />
-          <Tooltip
-            arrow
-            inverted
-            title={
-              isDownloadDisabled
-                ? DOWNLOAD_TOOLTIP_TEXT_DISABLED
-                : DOWNLOAD_TOOLTIP_TEXT_ENABLED
-            }
-            placement="top"
-          >
-            <StyledSpan>
-              <IconButtonBubble
-                onClick={handleDownloadClickOpen}
-                disabled={isDownloadDisabled}
-              >
-                {isDownloadDisabled ? (
-                  <StyledDownloadDisabledImage />
-                ) : (
-                  <StyledDownloadImage />
-                )}
-              </IconButtonBubble>
-            </StyledSpan>
-          </Tooltip>
+          <IconButton
+            onClick={handleCreateTreeClickOpen}
+            disabled={isDownloadDisabled}
+            svgDisabled={<StyledTreeBuildDisabledImage />}
+            svgEnabled={<StyledTreeBuildImage />}
+            tooltipTextDisabled={TREE_BUILD_TOOLTIP_TEXT_DISABLED}
+            tooltipTextEnabled={TREE_BUILD_TOOLTIP_TEXT_ENABLED}
+          />
+          <IconButton
+            onClick={handleDownloadClickOpen}
+            disabled={isDownloadDisabled}
+            svgDisabled={<StyledDownloadDisabledImage />}
+            svgEnabled={<StyledDownloadImage />}
+            tooltipTextDisabled={DOWNLOAD_TOOLTIP_TEXT_DISABLED}
+            tooltipTextEnabled={DOWNLOAD_TOOLTIP_TEXT_ENABLED}
+          />
         </DownloadWrapper>
       );
     }
 
     return (
       <>
         {tableData !== undefined && viewName === VIEWNAME.SAMPLES && (
-          <DownloadModal
-            sampleIds={checkedSamples}
-            failedSamples={failedSamples}
-            setDownloadFailed={setDownloadFailed}
-            isMetadataSelected={isMetadataSelected}
-            setMetadataSelected={setMetadataSelected}
-            isFastaSelected={isFastaSelected}
-            setFastaSelected={setFastaSelected}
-            isFastaDisabled={isFastaDisabled}
-            setFastaDisabled={setFastaDisabled}
-            tsvData={tsvDataMap(checkedSamples, tableData, headers, subheaders)}
-            open={open}
-            onClose={handleDownloadClose}
-          />
+          <>
+            <DownloadModal
+              sampleIds={checkedSamples}
+              failedSamples={failedSamples}
+              setDownloadFailed={setDownloadFailed}
+              isMetadataSelected={isMetadataSelected}
+              setMetadataSelected={setMetadataSelected}
+              isFastaSelected={isFastaSelected}
+              setFastaSelected={setFastaSelected}
+              isFastaDisabled={isFastaDisabled}
+              setFastaDisabled={setFastaDisabled}
+              tsvData={tsvDataMap(
+                checkedSamples,
+                tableData,
+                headers,
+                subheaders
+              )}
+              open={open}
```

Comment:
Not necessary to fix this one, but since there are now two modals, I might consider renaming this to something more descriptive. It _is_ distinct enough from the other one to tell them apart, but no longer gives necessary context on what is open to be a good stand alone var name.

Reply:
i for sure agree that this should be something more descriptive, I can add that now.